hooks/pre_commit/20-certs_update.rb

```diff
@@ -0,0 +1,20 @@
+if module_enabled?('certs')
+  if app_value(:certs_update_server_ca) && !module_enabled?('katello')
+    fail_and_exit("--certs-update-server-ca needs to be used with katello", 101)
+  end
+
+  if app_value(:certs_reset) && !app_value(:noop)
```

Comment:
I think it's safe to unset in noop. In fact, if you don't you'll get different results between noop and actual runs.
```suggestion
  if app_value(:certs_reset)
```

In a noop run the answers are not saved, so that's not a problem.

Reply:
Good point... updated